packages/libtreesitter/Makefile.patch

```diff
@@ -0,0 +1,12 @@
+diff --color -ur tree-sitter-old/Makefile tree-sitter-new/Makefile
+--- tree-sitter-old/Makefile	2021-06-30 06:18:39.000000000 +0600
++++ tree-sitter-new/Makefile	2021-07-04 21:29:33.621254194 +0600
+@@ -1,7 +1,7 @@
+ VERSION := 0.6.3
+ 
+ # install directory layout
+-PREFIX ?= /usr/local
++PREFIX ?= ${TERMUX_PREFIX}/usr/local
```

Comment:
Use just `PREFIX ?= @TERMUX_PREFIX@`, `${TERMUX_PREFIX}/usr/local` would expand to `/data/data/com.termux/files/usr/usr/local`, which is problematic since it has two /usr. System packages should also not be installed to usr/local, that folder is for stuff installed by the user locally

Reply:
Right I forgot to remove the rest

Reply:
@Grimler91 I'm actually wondering if makefile patch is needed . I think it's using `PREFIX` env set during build because of ` ?= ` . And PREFIX seems to set right . Should I remive the patch all togather ?

Reply:
If it compiles and installs to the correct place without it, then yeah, remove it